sdk/schema/executable.go

```diff
@@ -21,12 +21,12 @@ type Executable struct {
 	// (Optional) A URL to the documentation about this executable.
 	DocsURL *url.URL
 
-	// (Optional) Whether the exectuable needs authentication for certain args.
+	// (Optional) Whether the executable needs authentication for certain args.
 	NeedsAuth sdk.NeedsAuthentication
 }
 
 type CredentialUsage struct {
-	// The name of the credential to use in the executable.
+	// (Optional) The name of the credential to use in the executable. Mutually exclusive with `SelectFrom`.
 	Name sdk.CredentialName
 
 	// (Optional) The plugin name that contains the credential. Defaults to the current package. This can be used to
```

Comment:
Does "Mutually exclusive with `SelectFrom`" need to be added here too?

Reply:
Done, thanks!